torch_np/tests/numpy_tests/fft/test_pocketfft.py

```diff
@@ -40,13 +41,15 @@ def test_identity(self):
                             xr[0:i], atol=1e-12)
 
     def test_fft(self):
+
+        np.random.seed(1234)
         x = random(30) + 1j*random(30)
-        assert_allclose(fft1(x), np.fft.fft(x), atol=1e-6)
-        assert_allclose(fft1(x), np.fft.fft(x, norm="backward"), atol=1e-6)
+        assert_allclose(fft1(x), np.fft.fft(x), atol=2e-5)
```

Comment:
Had to bump the tolerance. Not much we can do I guess, this fall straight through to pytorch.